docs/misc/world-data.md

```diff
@@ -80,6 +80,10 @@ Example of expected VanillaTweaks share codes:
 VANILLATWEAKS_SHARECODE: MGr52E,tF1zL2,LnEDwT
 ```
 
+!!! note
+
+    Datapack names are all lower case. See https://vanillatweaks.net/assets/resources/json/1.21/dpcategories.json for a full list of 1.21 datapacks, and https://vanillatweaks.net/assets/resources/json/1.21/ctcategories.json for a full list of 1.21 crafting tweaks.

```

Comment:
As you found, it's just markdown, so makes it pretty easy to format. You can always preview the rendered docs via the PR checks link such as 

https://docker-minecraft-server--3287.org.readthedocs.build/en/3287/misc/world-data/#vanillatweaks

Speaking of which, being lengthy URLs, the URLs here wrap a bit funny on a mobile device (since that's where I'm reviewing this). Perhaps wrap those in markdown links like 

```suggestion
    Datapack names are all lower case. [See their spec](https://vanillatweaks.net/assets/resources/json/1.21/dpcategories.json) for a full list of 1.21 datapacks, and [their spec](https://vanillatweaks.net/assets/resources/json/1.21/ctcategories.json) for a full list of 1.21 crafting tweaks.
```